src/locales/po-files/openverse.pot

```diff
@@ -1868,7 +1868,6 @@ msgctxt "hero.aria.search-type"
 msgid "search type"
 msgstr ""
 
-#: src/components/LocaleSelector.vue:3
 msgctxt "hero.locale.label"
```

Comment:
The locale selector strings are still there in the `.pot` file. I guess if we delete them from the JSON file, they will be removed from `.pot`, too?